### ISSUE TITLE
Support for experimental vc4 driver in the kernel

### ISSUE THREAD

**Issue by agherzan** [resin-os/resinos#72]
_From @shaunmulligan on March 17, 2016 12:28_

The current kernel doesnt have the patch set to include the vc4 driver 
The required firmware is here:
https://github.com/raspberrypi/firmware/tree/master/modules/4.1.19%2B

_Copied from original issue: resin-os/resin-raspberrypi#7_

**Comment by lurch** [comment]
Is resinOS really the right place for 'experimental drivers'?

**Comment by agherzan** [comment]
You are right. In the meanwhile vc4 is not experimental anymore as being merged in rpi kernel but even when it was, we wanted to do it as a config option (that would not be default) for people to be able to switch on vc4 easily.

**Comment by shaunmulligan** [comment]
@agherzan I agree we shouldn't put this in as default until its completely stable and official. Only one user has ever raised it so far.
